medicaltorch/datasets.py

```diff
@@ -61,29 +63,38 @@ class SegmentationPair2D(object):
     """This class is used to build 2D segmentation datasets. It represents
     a pair of of two data volumes (the input data and the ground truth data).
 
-    :param input_filename: the input filename (supported by nibabel).
+    :param input_filename: the input filename list (supported by nibabel). For single channel, the list will contain 1
+                           input filename.
     :param gt_filename: the ground-truth filename.
+    :param metadata: metadata list related to images 1.  For single channel, the list will contain metadata related to
+                     to one image.
+    :param modality: contrast list related to images (e.g. T2w). For single channel, the list will contain 1 modality.
```

Comment:
Was just remembering that modality information can be retrieved from metadata --> so maybe we could use metadata info and avoid duplication. Does it sound good to you @andreanne-lemay ?

Reply:
Yes good idea!

Reply:
The contrast in metadata is under a different format than our usual contrast e.i. t2s instead of acq-inf_T2star. So we might need to add those keys to the dict. But I am not sure of the form of every modality in metadata.

Reply:
Not sure to answer the question but: Could we use `contrast_dct.json` like [here](https://github.com/neuropoly/ivado-medical-imaging/blob/master/ivadomed/loader.py#L302)?
We keep the modality as formated in the json, and map in further for our specific application.